src/containers/AppRoot.tsx

```diff
@@ -65,7 +64,7 @@ class AppRoot extends React.Component<Props> {
         await connectAndListen(publicIdentityFromStore.address)
         // eagerly get the balance
         const balance = await getBalanceInKiltCoins(
-          publicIdentityFromStore.address
+          publicIdentityFromStore.address,
```

Comment:
Would we need to change this with the update from the SDK? `identity.getPublicIdentity()` instead of saving the `publicIdentity` saving the identity directly.